docs/install/macos.mdx

```diff
@@ -9,99 +9,71 @@ title: Build on MacOS
 In order to compile PcapPlusPlus on MacOS please make sure you have the following components installed:
 
 1. [Xcode](https://developer.apple.com/xcode/) which contains all prerequisites required for PcapPlusPlus including gcc/g++ compiler and libpcap with all relevant header files
-2. Make sure you have Xcode Command Line Tools. You can install it by running the following command in Terminal:
-
-   ```shell
-   xcode-select --install
-   ```
-
-3. PcapPlusPlus is available for both **Intel** and **Apple silicon (M1)** processors. To build for Apple silicon please use the `--arm64` flag (please see below)
-4. **OPTIONAL** - install [Zstd](https://facebook.github.io/zstd/) if you wish to enable [PCAPNG streaming compression support](../features#readingwriting-pcapng-files-with-compression):
+2. PcapPlusPlus is available for both **Intel** and **Apple Silicon** processors. To build for a specific architecture you can use the `-DCMAKE_OSX_ARCHITECTURES` option, see more details below
```

Comment:
```suggestion
2. PcapPlusPlus is available for both **Intel** and **Apple Silicon** processors. To cross-compile for a different architecture you can use the `-DCMAKE_OSX_ARCHITECTURES` option, see more details below
```

Reply:
Fixed in https://github.com/PcapPlusPlus/pcapplusplus.github.io/pull/48/commits/cbb93cd88cf00bd702a3ec936eceba90f8bfc47f